weblate_web/templates/support.html

```diff
@@ -137,13 +137,13 @@ <h2 class="section-title bigger">{% trans "Mailing lists" %}</h2>
                 <form class="mailing" action="{% url 'subscribe' name='hosted' %}" method="post">
                     {% csrf_token %}
                     <label>{% trans "E-mail" %}</label>
-                    <input type="email" name="email" class="mailing-text" placeholder="{% trans "Your e-mail..." %}" />
+                    <input type="email" name="email" class="mailing-text" placeholder="{% trans "Your e-mail…" %}" />
                     <input type="submit" class="mailing-submit" value="{% trans "Subscribe" %}" />
                 </form>
                 <ul>
                     <li>{% trans "<strong>Low traffic</strong> mailing list with announcements about premium Weblate services" %}</li>
-                    <li>{% trans "The list is moderated — only selected users can post announcements" %}</li>
-                    <li>{% blocktrans with url="https://lists.cihar.com/hyperkitty/list/hosted-weblate-announce@lists.cihar.com/" %}Browse <a href="{{ url }}">mailing list archives</a> to find answers{% endblocktrans %}</li>
+                    <li>{% trans "The list is moderated — only select users can post announcements" %}</li>
```

Comment:
Really? This doesn't sound right to me.

Reply:
It is a bit subtle, but "a select few" means the few that are selected, have some distinguishing feature, as opposed to "a selected few" could mean a narrow random assortment.
https://www.dailywritingtips.com/select-vs-selected/